plone/app/discussion/profiles/default/rolemap.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0"?>
 <rolemap>
     <permissions>
+        <permission name="View comments" acquire="True">
```

Comment:
@avoinea  You can add the permission to the rolemap.xml so the previous behavior  (comments visible for anonymous) is added again, but you would also need an upgrade step to apply this rolemap in existing sites when you upgrade. 

Not sure where is has to go, but I assume plone.app.upgrade.

Reply:
@fredvd I see there is an `upgrade.zcml` within this package. Thus, I think it should go first here, @razvanMiu ^

Reply:
@fredvd I see @razvanMiu already added the upgrade step here. 

I don't know about plone.app.upgrades, when and how it should go there :thinking: @mauritsvanrees @jensens @ale-rt

Reply:
`plone.app.upgrades` can have a step loading the step from here.

Reply:
`plone.app.discussion:default` is in the `ADDON_LIST` in `CMFPlone.MigrationTool`, so an upgrade is automatically applied during `@@plone-upgrade`.
I just tested the upgrade step here. Indeed needed. Before running it, anonymous cannot see comments. Afterwards, you can.